lib/router/router.js

```diff
@@ -77,6 +78,7 @@ export default class Router extends EventEmitter {
 
   async reload (route) {
     delete this.components[route]
+    await reloadIfPrefetched(route)
```

Comment:
We don't need to do anything for normal HMR since the page gets updated via HMR. 

We need to do this, because we are removing the page from the cache and loading a error page instead.